packages/atomic/project.json

```diff
@@ -1 +1,264 @@
+        "{projectRoot}/scripts/split-locales.mjs",
+        "{projectRoot}/scripts/copy-dayjs-locales.mjs",
+        "{projectRoot}/src/locales.json"
+      ],
+      "outputs": [
+        "{projectRoot}/src/generated/availableLocales.json",
+        "{projectRoot}/src/generated/dayjs-locales-data.ts",
+        "{projectRoot}/src/components/search/atomic-search-interface/lang"
+      ],
+      "executor": "nx:run-commands",
+      "options": {
+        "commands": [
+          "node ./scripts/create-generated-folder.mjs",
+          "node ./scripts/split-locales.mjs",
+          "node ./scripts/copy-dayjs-locales.mjs"
+        ],
+        "parallel": false,
+        "cwd": "packages/atomic"
+      }
+    },
+    "cached:build:stencil": {
+      "dependsOn": ["^build", "cached:build:locales"],
+      "inputs": ["^source", "source"],
+      "outputs": [
+        "{projectRoot}/dist",
+        "{projectRoot}/www",
+        "{projectRoot}/docs",
+        "{projectRoot}/loader",
+        "{projectRoot}/src/components.d.ts"
+      ],
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "node --max_old_space_size=6144 ../../node_modules/@stencil/core/bin/stencil build",
+        "cwd": "packages/atomic"
+      }
+    },
+    "cached:validate:definitions": {
+      "dependsOn": ["^build", "cached:build:stencil"],
+      "inputs": ["{projectRoot}/dist/types/components.d.ts"],
+      "outputs": [],
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "tsc --noEmit --esModuleInterop --skipLibCheck ./dist/types/components.d.ts",
+        "cwd": "packages/atomic"
+      }
+    },
+    "cached:validate:storybook": {
+      "dependsOn": ["^build"],
+      "inputs": ["stories"],
+      "outputs": [],
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "lit-analyzer src/components/**/*.stories.tsx",
+        "cwd": "packages/atomic"
+      }
+    },
+    "cached:build:storybook:assets-list": {
+      "dependsOn": ["cached:build:stencil"],
+      "inputs": [
+        "{projectRoot}/scripts/list-assets.mjs",
+        "{projectRoot}/dist/atomic/assets/**"
+      ],
+      "outputs": ["{projectRoot}/docs/assets.json"],
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "node scripts/list-assets.mjs",
+        "cwd": "packages/atomic"
+      }
+    },
+    "cached:build:storybook": {
+      "dependsOn": [
+        "cached:build:stencil",
+        "cached:build:storybook:assets-list"
+      ],
+      "inputs": [
+        "^source",
+        "source",
+        "stories",
+        "{projectRoot}/.storybook/**",
+        "{projectRoot}/docs/assets.json"
+      ],
+      "outputs": ["{projectRoot}/storybook-static"],
+      "executor": "nx:run-commands",
+      "options": {
+        "commands": ["build-storybook", "ncp dist/ storybook-static/"],
+        "parallel": false,
+        "cwd": "packages/atomic"
+      }
+    },
+    "build": {
```

Comment:
for "top-level" tasks like "build" or "test", I decided to use the `nx:noop` executor with dependencies so that we can simply add as dependencies the list of tasks we need to be run, and Nx will figure out in what order they need to be run.